### DISCUSSION TITLE
[30 Giugno 2023] - Cache

### DISCUSSION THREAD

**Discussion by rimaout** [CS-Swap/Architettura-degli-Elaboratori#34] [Cache]
### Traccia Esercizio

![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/6f5c3fc6-ef86-4830-bdb6-1e9705039726)

---
>**[Altre Soluzioni](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/63)**

**Comment by thegeek-sys** [comment]
<img width="1082" alt="Screenshot 2024-06-04 alle 18 38 48" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/65cb7800-afb4-4fa8-b539-8baaa5e72a55">

**Comment by notedo** [comment]
![ESERCIZI - Architettura-90](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/35872b06-1022-40d8-b522-76b3f64e7b97)
![IMG_0519](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/b7d8e8fa-9f75-4e7a-bda4-e6bf5940aa38)

**Comment by rimaout** [comment]
Versione con tutte le formule utilizzate

<img width="773" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/c06667bd-65cf-4d7a-b3f4-19ae8e0f780c">
